### PR TITLE
models created

### PR DESCRIPTION
**Pull request by SrijanShovit**

### CODE REVIEW

File: models/Individual.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        trim:true,
+    },
+    email:{
+        type: String,
+        required: true,
+        unique:true,
+        trim:true,
+        validate: [ isEmail, 'Please provide a valid email ID' ]
+    },
+    password:{
```
Comment (rishabhborn2win) [inline models/Individual.js:18]: create a hashing function for saving the encrypted password before saving the detail in the schema

Comment (SrijanShovit) [reply models/Individual.js:20]: created

**Review by PalakKumari — CHANGES_REQUESTED** [review]
In the event schema, the organiserId will be referenced by organisation as well as the individual schema( organiser of an event can be an individual as well). Change the ref there.
In the individual schema, set 2 fields for LinkedIn and Facebook in place on one social media field.
In otp schema remove the isOffline field and add isUsed field as in the diagram.

**Comment by SrijanShovit** [comment]
done

**Review by PalakKumari — APPROVED** [review]

**Pull request merged**